hack/build-image/Dockerfile

```diff
@@ -14,6 +14,7 @@
 
 FROM golang:1.13
 
+ENV GO111MODULE=on
```

Comment:
I actually don't know that we need this set to `on` anymore, given it's on by default in 1.14. Did you find that it was necessary as you were testing this out during your changes?

Reply:
probably not needed, let me try things out without it - thanks for catching.

Reply:
looks like it's still needed to do `go get` with an `@<version>` suffix, since we're doing that outside of a module.

Reply:
Specifically, for:

```
RUN go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4
```